concourse/pipelines/dev-image-build.yaml

```diff
@@ -366,3 +434,53 @@ jobs:
       publish_version: ((.:publish-version))
       wf: "debian/debian_10_worker.publish.json"
       environment: "prod"
+- name: publish-to-staging-rhel-7-metal-dev
```

Comment:
These will overwrite the non-dev publish images since they use the same workflow.

Reply:
Oh, right. We need to a separate image family.

Reply:
I create a `rhel-7-metal-dev.publish.json` and `rhel-8-metal-dev.publish.json` in linux_dev